code/nxt/src/movement.c

```diff
@@ -110,14 +115,10 @@ int8_t get_speed_by_distance(int8_t distance, char axis) {
         return 0;
     }
 
-    uint8_t lower_bound = get_required_power(axis, distance >= 0);
-    uint8_t range = 50;
-
-    // if distance is negative, then MOTOR_SPEED_LOWER_BOUND should be negative,
-    // otherwise we don't get a value in the expected range
-    int8_t actual_lower_bound = lower_bound * ((distance >= 0) ? 1 : -1);
-
-    return -((distance * range) / MAX_INPUT_VALUE + actual_lower_bound);
+    int8_t lower_bound = get_required_power(axis, distance >= 0) * ((distance >= 0) ? 1 : -1);
+    // The first magic numbers below regulates speed based on distance
+    // The second number is used to regulate the speed at the center
+    return distance / 5 + lower_bound - (lower_bound / 5);
```

Comment:
Should this actually be magic number here tho? Shouldn't we use Constants instead?